deu/match-cleared.vbs

```diff
@@ -911,6 +911,7 @@ IF resolution_status = "CC-Overpayment Only" or HC_OP_checkbox = CHECKED THEN
 END IF
 CALL write_bullet_and_variable_in_case_note("Other Notes", other_notes)
 CALL write_variable_in_case_note("----- ----- ----- ----- ----- ----- -----")
+CALL write_variable_in_case_note(worker_signature)
```

Comment:
Do....loop on lines 85-117 does not have password handling. While this line was only added to the CASE/NOTE on this script as part of the issue, the mandatory handling associated with this issue is affected by not having a complete do....loop series. Please update.